src/chrome/playSound.js

```diff
@@ -0,0 +1,20 @@
+import { SOUND_URL, SOUND_REPEAT } from '../constants';
+
+// Get sound options and play if enabled
+
+export const getAndPlaySoundNotificationOption = (store, mainAudioPlayer) => {
```

Comment:
at first, I started off simple method... then got frustrated w/ testing suite..   but thanks for all the tips..  

definitely, I'll rather to pass options as arguments.   but I'm trying to figure out how to use sinon.spy on audioPlayer.  That's why I went ahead passing it into argument.